src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -778,6 +778,14 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
         mapped_table = (fi_addr_t *) av_attr.map_addr;
         for (i = 0; i < size; i++) {
             MPIDI_OFI_AV(&MPIDIU_get_av(0, i)).dest = mapped_table[i];
+#if MPIDI_OFI_ENABLE_RUNTIME_CHECKS
+            MPIDI_OFI_AV(&MPIDIU_get_av(0, i)).ep_idx = 0;
+#else /* This is necessary for older GCC compilers that don't properly detect
+       * elif statements */
+#if MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS
+            MPIDI_OFI_AV(&MPIDIU_get_av(0, i)).ep_idx = 0;
+#endif
+#endif
```

Comment:
This can just be:

    if (MPIDI_OFI_ENABLE_RUNTIME_CHECKS && MPIDI_OFI_ENABLE_SCALABLE_ENDPOINTS)

Can't it?